src/frontend/templates/pages/account.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html' %}
+
+{% load i18n %}
+{% load custom_formatting %}
+{% load custom_translation %}
+
+{% block title %}{% trans "KataGo - Account" %}{% endblock %}
+
+
+{% block content %}
+
+<h2 class="title">{% trans "KataGo - Account" %}</h1>
+
+{% url 'account_change_password' as password_url %}
+<a href="{{password_url}}">
+  <button class="button is-warning">
+    {% trans "Change password" %}
+  </button>
+</a>
+
+{% url 'account_email' as email_url %}
```

Comment:
Does the account_email exist ?

Reply:
Yes, it is defined by django allauth:
https://django-allauth.readthedocs.io/en/latest/views.html